### ISSUE TITLE
Vscode April 2023 1.78.1 sometimes lags when clicking mouse or rolling the mouse wheel to scroll

### ISSUE THREAD

**Issue by jhancibo** [microsoft/vscode-jupyter#13644]
### Applies To

- [ ] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

Vscode April 2023 1.78.2 sometimes lags when clicking the mouse or rolling the mouse wheel to scroll.

### VS Code Version

1.78.2

### Jupyter Extension Version

v2023.4.1011241018

### Running Jupyter locally or remotely?

None

**Comment by jhancibo** [comment]
Often the vscode crashes.
The window terminated unexpectedly (reason: 'crashed', code: '5')
We are sorry for the inconvenience. You can reopen the window to continue where you left off.

**Comment by DonJayamanne** [comment]
I'm very sorry for missing this issue,
Are you still running into this issue, I would like to get you unblocked from this.

**Comment by jhancibo** [comment]
It's fixed, thanks!

**Comment by jhancibo** [comment]
Actually the new verion of VSCODE 1.81.1  (Universal) broken and super slow
down the jupyter notebook interactive cells operation, such as simple as
df.head(), could you guys fix it? If not, when is the next release? Thanks!

**Comment by DonJayamanne** [comment]
> down the jupyter notebook interactive cells operation, such as simple as
df.head(), could you guys fix it? If not, when is the next release? Thanks!

What is slow?
is it scrolling, or the execution?
To make it easier to triage issues please could you create separate Github issues for different problems, else its a little diffiicult to keep track of the problems in the issue.
Please create a new issue for this problem you are now running into.

> If not, when is the next release? Thanks!

Some time this week


> It's fixed, thanks!

Closing this issue as you have indicated its fixed.

**Comment by jhancibo** [comment]
Hi Don,
I understand you guys have priority, but could you take a look at this new
ticket yourself https://github.com/microsoft/vscode-jupyter/issues/14283
instead of your colleague? Without fixing this issue soon, all of our data
science team members are switching to Jupyterlab on a daily basis.

Thanks,
James
